### ISSUE TITLE
Name meaning in french

### ISSUE THREAD

**Issue by raphaelbastide** [epilys/gerb#9]
Just so you know, and this may be not a problem at all, but French speaking people may quickly think about “gerbe” meaning “puke” in french.

**Comment by epilys** [comment]
Haha I didn't know that. I don't mind as long as it's not something offensive/derogatory. (as mentioned in the README it's the reconstructed Proto-Indo-European root of the word "curve"). But it's not a final name, if and when it becomes actually useful it should probably have a better name too.

**Comment by raphaelbastide** [comment]
Yeah, definitively not offensive. The good thing is that it will stick in mind :brain:!
